misc/update-repo.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: MickLesk
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+function header_info {
+  clear
+  cat <<"EOF"
+   __  __          __      __          ____                 
+  / / / /___  ____/ /___ _/ /____     / __ \___  ____  ____ 
+ / / / / __ \/ __  / __ `/ __/ _ \   / /_/ / _ \/ __ \/ __ \
+/ /_/ / /_/ / /_/ / /_/ / /_/  __/  / _, _/  __/ /_/ / /_/ /
+\____/ .___/\__,_/\__,_/\__/\___/  /_/ |_|\___/ .___/\____/ 
+    /_/                                      /_/            
+EOF
+}
+
+set -eEuo pipefail
+BL=$(echo "\033[36m")
+RD=$(echo "\033[01;31m")
+GN=$(echo "\033[1;92m")
+CL=$(echo "\033[m")
+
+header_info
+echo "Loading..."
+NODE=$(hostname)
+
+function update_container() {
+  container=$1
+  os=$(pct config "$container" | awk '/^ostype/ {print $2}')
+
+  if [[ "$os" == "ubuntu" || "$os" == "debian" ]]; then
+    echo -e "${BL}[Info]${GN} Checking /usr/bin/update in ${BL}$container${CL} (OS: ${GN}$os${CL})"
+
+    if pct exec "$container" -- [ -e /usr/bin/update ]; then
+      if pct exec "$container" -- grep -q "community-scripts/ProxmoxVE" /usr/bin/update; then
+        echo -e "${RD}[No Change]${CL} /usr/bin/update is already up to date in ${BL}$container${CL}.\n"
+      elif pct exec "$container" -- grep -q "tteck" /usr/bin/update; then
```

Comment:
I found I had to change to it from `grep -q` to `grep -qv` for the update to work for me because if it does not match `tteck` then it should return that no changes made.

Reply:
BROTHER